sdcm/utils/k8s.py

```diff
@@ -429,6 +429,28 @@ def wait_cluster_is_ready():
 
         wait_cluster_is_ready()
 
+    @staticmethod
+    def wait_for_pods_running(kluster, total_pods: Union[int, Callable], readiness_timeout: float, namespace: str,
+                              sleep: int = 10):
```

Comment:
maybe rename `sleep` to `sleep_between_retries` to reflect the purpose?

Reply:
it's copy-paste from `wait_for_pods_readiness`, and maps to `retrying.sleep_time` 
since it's not used in both, it can be renamed

Reply:
changed